common/bg.po

```diff
@@ -2,13 +2,16 @@
 # Copyright (C) 2001-2022 Celestia Development Team
 # This file is distributed under the same license as the celestia package.
 #
+# Translators:
```

Comment:
can you remove this? already listed below

Reply:
Done.